internal/dao/entities/job.go

```diff
@@ -0,0 +1,50 @@
+package entities
+
+import (
+	"time"
+
+	"casper-dao-middleware/pkg/casper/types"
+)
+
+type JobStatus byte
+
+const (
+	JobStatusCreated JobStatus = iota + 1
+	JobStatusSubmitted
+	JobStatusCancelled
+	JobStatusDone
+	JobStatusRejected
```

Comment:
Let's define those as job status IDs because `JobStatus` is an implicit entity in the domain and we should provide a getter for it `GET /job-statuses`